api/app/controllers/spree/api/orders_controller.rb

```diff
@@ -116,6 +109,15 @@ def normalize_params
         params[:order][:bill_address_attributes] = params[:order].delete(:bill_address) if params[:order][:bill_address].present?
       end
 
+      # @api public
+      def determine_order_user
+        if order_params[:user_id]
```

Comment:
I tend to add `#present?` incase the param is an empty string like if `order_params[:user_id] = ''`.